innovation_sweet_spots/analysis/wrangling_utils.py

```diff
@@ -269,6 +269,62 @@ def convert_deal_currency_to_gbp(
             # If nothing to convert, copy the nulls and return
             df[converted_column] = df[amount_column].copy()
         return df
+      
+    def get_funding_round_investors(self, funding_rounds: pd.DataFrame) -> pd.DataFrame:
+        """
+        Gets the investors involved in the specified funding rounds
+
+        Args:
+            Dataframe with organisations specified by 'org_id' and 'name', and
+            all their funding rounds (deals). Some of the important columns include:
```

Comment:
I noticed after approving that the Args section here doesn't refer to the variable and the sentence isn't complete.